Assets/QA/Tests/XRDeviceActions/GearVR/GearVRHandedness.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using UnityEngine.UI;
+using UnityEngine.Experimental.Input;
+using UnityEngine.Experimental.Input.Plugins.XR;
+
+public class GearVRHandedness : MonoBehaviour
+{
+    public Text statusText;
+
+    // Update is called once per frame
+    void Update()
```

Comment:
This should be easier to check:
bool hasLeft = (XRController.leftHand as GearVRTrackedController) != null;
bool hasRight = (XRController.leftHand as GearVRTrackedController) != null;

Then do the string assignment.
Not a blocker though.